game/src/common/mod.rs

```diff
@@ -116,7 +116,7 @@ impl CommonState {
         };
         if !keys.is_empty() {
             osd.append(Line("   Hotkeys: "));
-            for (idx, key) in keys.into_iter().enumerate() {
```

Comment:
Not sure this makes sense. We don't use `keys` again, so why not consume it? `key.describe` consumes the key, and even though it's `Copy`, why would we want to? Negligible from a perf perspective, but marginally less work to avoid the copy.

Reply:
I think it's similar to preferring `let` over `let mut`, or `&T` over `&mut T`. I would expect that the generated code is the same for both variants.